tests/roles/backend_services/tasks/ospdo_backend_services.yaml

```diff
@@ -6,7 +6,7 @@
 
 - name: get OSPdO storage storageClass
   ansible.builtin.shell: |
-    oc get -n {{ org_namespace }} pvc openstackclient-hosts -o jsonpath='{.spec.storageClassName}'
+    oc get pvc openstackclient-hosts -o jsonpath='{.spec.storageClassName}'
```

Comment:
Does this one actually need the `-n`? I think we want to run it in the OSPdO namespace and not the destination RHOSO namespace?